blog/articles/event-drive-app-knative-eventing-kogito.md

```diff
@@ -0,0 +1,130 @@
+---
+Title: 'Orchestrating Events with Knative and Kogito'
+Author: Ricardo Zanini and Tihomir Surdilovic
+Author handle: https://twitter.com/zaninirica, https://twitter.com/tsurdilo
+Date: ''
```

Comment:
Note: Remember to add a date when we merge :slightly_smiling_face:

Reply:
Which date should I add? Maybe today?